packages/server/src/addresses/domain/UpdateAddressHandler.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+
+import { Handler } from '../../shared';
+import { Address, AddressDocument } from '../database';
+
+interface UpdateAddressRequest {
+  readonly id: string;
+  readonly street: string;
+  readonly streetNumber: string;
+  readonly apartmentNumber: number;
+  readonly floor: number;
+  readonly city: string;
+  readonly postcode: string;
+  readonly longitude: number;
+  readonly latitude: number;
+}
+
+@Injectable()
+class UpdateAddressHandler implements Handler<UpdateAddressRequest, null | undefined> {
+  constructor(@InjectModel(Address.name) private addressModel: Model<AddressDocument>) {}
+
+  async exec(req: UpdateAddressRequest): Promise<null | undefined> {
+    const result = await this.addressModel.findOneAndUpdate(
+      { _id: req.id },
+      {
+        street: req.street,
+        streetNumber: req.streetNumber,
+        apartmentNumber: req.apartmentNumber,
+        floor: req.floor,
+        city: req.city,
+        postcode: req.postcode,
+        longitude: req.longitude,
+        latitude: req.latitude,
+      },
```

Comment:
```suggestion
      _.pick(req, 'street', 'streetNumber', 'apartmentNumber', 'floor', 'city', 'postcode', 'longitude', 'latitude'),
```

Może coś takiego, żeby nie było tyle powtarzania się?

Reply:
To, może łatwiej w drugą stronę 😄:
```suggestion
_.omit(req,'id')
```
lub
```suggestion
{...req, id: undefined }
```

Reply:
Prawda, martwiłem się requestami z dodatkowymi kluczami dopisanymi przez usera ale to jest już filtrowane wcześniej 👍